src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -43,6 +43,12 @@
 
 BLACKLISTED_MODS = ['context', 'shell', 'documentdb']
 
+# Can use operators such as lt, le, eq, ne, ge, gt from
+# https://docs.python.org/3/library/operator.html#mapping-operators-to-functions
+BLACKLISTED_EXTENSIONS = {
+    'image-copy-extension': (le, '0.0.4'),
```

Comment:
Would remove this before merging as we don't actually want to blacklist this extension.